src/polyswarm_api/__init__.py

```diff
@@ -371,19 +381,24 @@ def _fix_result(self, result):
         with open(to_scan, 'rb') as fobj:
             return await self.scan_fileobj(fobj, os.path.basename(to_scan))
 
-    async def search_hash(self, to_scan, hash_type='sha256'):
+    async def search_hash(self, hash_string):
```

Comment:
we want to force the user to specify the type of hash. And if they do not specify the type of hash then we will assume sha256. So, I need you to re-add the "hash_type" arg to all of these functions. Then update the docs to tell the user to specify the hash type on the command line. And if no hash type is provided, then we will assume sha256.